### DISCUSSION TITLE
Plugin collection name conflicts with existing collection name "comments"

### DISCUSSION THREAD

**Discussion by RajNandigalla** [VirtusLab-Open-Source/strapi-plugin-comments#27] [Q&A] [answered]
Plugin collection name conflicts with existing collection name "comments"
Anyway to customize this ?

**Comment by cyp3rius** [comment, marked as answer]
Hi @RajNandigalla, I think there are two possibilities:

1. Make an [extensions](https://strapi.io/documentation/developer-docs/latest/concepts/customization.html#plugin-extensions) (at least for routes.json) and define them differently.
2. If you would like to use comments plugin make a migration of your own content type, to the plugins comments content type.
